aocd/utils.py

```diff
@@ -209,26 +215,39 @@ def _cli_guess(choice, choices):
     os.system("color")  # hack - makes ANSI colors work in the windows cmd window
 
 
-def colored(txt, color):
+def colored(txt: str, color: Optional[str]) -> str:
     if color is None:
         return txt
     code = _ansi_colors.index(color.casefold())
     reset = "\x1b[0m"
     return f"\x1b[{code + 30}m{txt}{reset}"
 
+if sys.version_info >= (3, 10):
+    from importlib.metadata import EntryPoints
```

Comment:
This is very similar to the example parser loading code, and I think the common parts should be factored out into a `compat.py` helper function.